app/pastes/pasteRepository.js

```diff
@@ -15,6 +17,11 @@ class PasteRepository {
         }
 
         const result = await this.findOne({ id });
+        // added text here, not in findAll method, because findAll seems like a general search for pastes and
+        // we get text only when we open specific paste.
+        // Also here because i was thinking how to implement it in findAll and it was shit
```

Comment:
No, it should not. PasteRepository is not a place to fetch actual paste content:
1. We cannot be sure what do we search paste for. What if I use this method only to ensure the paste exists? Then content is redundant, and also, it may weight a lot of ram (useless)
2. We should fetch content only when we know it is for read-only purpose (e.g. usecases)